pkg/config/config_test.go

```diff
@@ -41,15 +42,15 @@ func TestAcornDNSStates(t *testing.T) {
 			// acornDNS is explicitly disabled, expect no clusterDomain to be returned
 			name: "acornDNS disabled expect no clusterdomains",
 			conf: &apiv1.Config{
-				AcornDNS: &[]string{"disabled"}[0], // hack for one-liner string pointer
+				AcornDNS: z.P("disabled"), // hack for one-liner string pointer
```

Comment:
nit: you can remove the comment here

```suggestion
				AcornDNS: z.P("disabled"),
```

Reply:
done